nodebuilder/node_test.go

```diff
@@ -125,34 +123,3 @@ func StartMockOtelCollectorHTTPServer(t *testing.T) (string, func()) {
 	server.EnableHTTP2 = true
 	return server.URL, server.Close
 }
-
-func TestEmptyBlockExists(t *testing.T) {
```

Comment:
That seems unrelated. Why? Even if it's not needed, let's keep PRs focused on their goal 🙏🏻

Reply:
Unnecessary test, that is tricky to fix. Removal is the best solution here

Reply:
Ok, agreed